src/component/GamePlay/GamePlay.jsx

```diff
@@ -74,6 +81,21 @@ function GamePlay({ favorites, setFavorites, errorMessage }) {
       .catch((error) => console.error("Error fetching predator data:", error));
   }
 
+  function fetchPreyData() {
+    const id = currentAnimal?.id;
+    fetch(
+      `http://localhost:3001/api/v1/animals/${id}/relationships?prey=true`
```

Comment:
This is good for now. A great place for refactoring later to DRY, however it works and it was a good call.